src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -365,43 +428,7 @@ function FloatingActionButtonAndPopover(
                         text: translate('sidebarScreen.fabNewChat'),
                         onSelected: () => interceptAnonymousUser(Report.startNewChat),
                     },
-                    ...(selfDMReportID
-                        ? [
-                              {
-                                  icon: getIconForAction(CONST.IOU.TYPE.TRACK),
-                                  text: translate('iou.trackExpense'),
```

Comment:
Where did you move this logic?

Reply:
I moved to a variable called `expenseMenuItems` that is now responsible of returning the expense menu items